server/src/lib.rs

```diff
@@ -138,9 +138,18 @@ pub fn initialize_state_actor(
     repository.add(&mut block_memory, genesis_block);
 
     let mut state = InMemoryState::new();
-    let (genesis_changes, table_changes) = genesis::init(&genesis_config, &state);
+    let (genesis_changes, table_changes) = {
```

Comment:
Is this more readable instead?
```rust
#[cfg(test)]
let (genesis_changes, table_changes) = moved_genesis_image::load();
#[cfg(not(test))]
let (genesis_changes, table_changes) = moved_genesis::build(&moved_genesis::MovedVm, &genesis_config, &state);
    
```

Reply:
I think it's more readable to have only one place where the variable is declared as it is currently.